multinode_runfabtests.sh

```diff
@@ -37,6 +37,63 @@ run_dgram_pingpong_with_b()
     return $ret
 }
 
+run_rdm_tagged_bw_fork()
```

Comment:
This function is a duplicate of `run_dgram_pingpong_with_b`, can we come up we with a common function for both?

Reply:
I think we can. But my concern is that we might need many extra parameters, like the command that we would like to run, whether the test is expected to pass or fail, etc.

Reply:
That is still preferred than duplicating a whole bunch of code.

Reply:
Ok. And I will add comments regarding how to call the new function as well.

Reply:
Now that we have `run_test_with_expected_ret`. Do we still need this function?

Reply:
Also the function `run_dgram_pingpong_with_b` can be removed. Right?